fastlane/Fastfile

```diff
@@ -21,6 +21,9 @@ files_with_version_number = {
   './docs/index.html' => ['url=https://sdk.revenuecat.com/android/{x}/index.html'],
   './gradle.properties' => ['VERSION_NAME={x}'],
   './.version' => ['{x}'],
+  './examples/MagicWeather/gradle/libs.versions.toml' => ['purchases = "{x}"'],
+  './examples/MagicWeatherCompose/gradle/libs.versions.toml' => ['purchases = "{x}"'],
+  './examples/CustomEntitlementComputationSample/gradle/libs.versions.toml' => ['purchases = "{x}"'],
```

Comment:
This means that we will also update to `-SNAPSHOT` and other prerelease modifiers versions... I thought about updating to the latest stable (which would generally be the previous minor version without prerelease modifiers), but then, I think it's ok to use the latest snapshot?

There is a gap however, when we bump the version to the next snapshot, where that version hasn't been deployed, and this will fail.

Reply:
I changed it so we only build the sample apps in `main` after the deploy-snapshot job is done... Not sure if that would be enough time to get it to be available when building the sample apps immediately after... Again, this will only matter the first time a snapshot version is deployed, the following times it would just use the previous deployment.

Reply:
I think it's fine, we can always wait and rerun